src/cls/IPM/Utils/LegacyCompat.cls

```diff
@@ -0,0 +1,34 @@
+Include %IPM.Formatting
+
+Class %IPM.Utils.LegacyCompat
+{
+
+/// This method updates the superclasses of a list of classes that have been loaded but not compiled.
+ClassMethod UpdateSuperclassAndCompile(ByRef tClasses, ByRef qSpec = "") As %Status
+{
+    Set tName = "" 
+    Set tToCompile = ""
+    For {
+        Set tName = $ORDER(tClasses(tName))
+        If (tName = "") { Quit } 
```

Comment:
Style nit: Best to have quit and continue on new lines. 1 command per line.

Reply:
Done.